crates/paralegal-policy/src/context.rs

```diff
@@ -29,6 +29,16 @@ pub type FunctionId = DefId;
 type MarkerIndex = HashMap<Marker, Vec<(DefId, MarkerRefinement)>>;
 type FlowsTo = HashMap<ControllerId, CtrlFlowsTo>;
 
+/// Enum for identifying an edge type (data, control or both)
+pub enum EdgeType {
+    /// Only consider dataflow edges
+    Data,
+    /// Only consider control flow edges
+    Control,
+    /// Consider both types of edges
+    DataOrControl,
```

Comment:
Is the better naming here maybe `DataAndControl` since it considers both at the same time?